packages/components/src/helperMessage/src/HelperMessage.tsx

```diff
@@ -0,0 +1,64 @@
+import { type StyledComponentProps, useStyledSystem } from "@hopper-ui/styled-system";
+import { forwardRef, type ForwardedRef, type CSSProperties } from "react";
+import clsx from "clsx";
+import { HelperMessageContext } from "./HelperMessageContext.ts";
+import { useContextProps } from "react-aria-components";
+import { InfoIcon } from "@hopper-ui/icons";
+import { type TextProps, Text } from "../../Text/index.ts";
+import { cssModule } from "../../utils/src/cssModule.ts";
+
+import styles from "./HelperMessage.module.css";
+
+export const GlobalHelperMessageCssSelector = "hop-HelperMessage";
+export interface HelperMessageProps extends StyledComponentProps<Omit<TextProps, "size">> {
+    /**
+     * Whether or not to show the helper message icon.
+     * @default true
+     */
+    showInfoIcon?: boolean;
```

Comment:
i know i'm the one that suggestion showInfoIcon, but i'd reverse it to hideInfoIcon. for boolean properties, we want to to have customer set it to true.

the guidelines would be: 
- Name boolean props that show or hide an option starting with show or hide depending on the default value.
- The default value for most boolean props should be false. Users shouldn’t typically be turning off options, but turning them on.